client/src/App.jsx

```diff
@@ -1,24 +1,30 @@
 import React from "react";
+import ReviewList from "./ReviewList.jsx";
 
 class App extends React.Component {
   constructor() {
     super();
     this.state = {
-      hi: true
+      reviews: []
     };
   }
 
   componentDidMount() {
     let randomProductId = Math.floor(Math.random() * 100);
     fetch(`/api/${randomProductId}/reviews/`)
       .then(response => response.json())
-      .then(data => console.log(data));
+      .then(data =>
+        this.setState({
+          reviews: data
+        })
+      );
```

Comment:
Should you include a catch statement to handle errors? At least logging to the console.